### ISSUE TITLE
How do you use this?

### ISSUE THREAD

**Issue by bonelifer** [Ashymad/kpxcnm#2]
How do you use this?

**Comment by Ashymad** [comment]
Yeah, there's no documentation, however it is just a straight up translation of the native messaging API https://github.com/keepassxreboot/keepassxc-browser/wiki/Protocol-V2-draft
As an exmplae you can look at my dotfiles https://github.com/Ashymad/Dotfiles/blob/notwobug/astroid/.config/astroid/get_pass.py
